tests/download_emulators.sh

```diff
@@ -3,6 +3,10 @@ SITE="https://firmware.corp.sldev.cz/upgrade_tests/"
 cd "$(dirname "$0")"
 
 # download all emulators without index files, without directories and only if not present
-wget --no-verbose --no-clobber --no-parent --cut-dirs=2 --no-host-directories --recursive --reject "index.html*" -P emulators/ $SITE
+wget -e robots=off --no-verbose --no-clobber --no-parent --cut-dirs=2 --no-host-directories --recursive --reject "index.html*" -P emulators/ $SITE
 
 chmod u+x emulators/trezor-emu-*
+
+if [[ `uname -a` =~ "NixOS" ]]; then
```

Comment:
Maybe just replace with `[ -f /etc/NIXOS ];`

Reply:
Fixed, surprisingly it failed on CI anyway.